packages/jellyfish-api-core/__tests__/category/masternode/getMasternodeBlock.test.ts

```diff
@@ -0,0 +1,143 @@
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import { ContainerAdapterClient } from '../../container_adapter_client'
+import { RpcApiError } from '../../../src'
+
+describe('Masternode', () => {
+  const container = new MasterNodeRegTestContainer()
+  const client = new ContainerAdapterClient(container)
+
+  beforeAll(async () => {
+    await container.start()
+    await container.waitForReady()
+    await container.waitForWalletCoinbaseMaturity()
+  })
+
+  afterAll(async () => {
+    await container.stop()
+  })
+
+  it('should getMasternodeBlocks with ID', async () => {
```

Comment:
`should getMasternodeBlocks with id`. ID should be lowercase like how you declare in the rest of the tests